Makefile

```
@@ -0,0 +1,23 @@
+CHART_NAME_1 ?= cray-vpa
+CHART_PATH ?= kubernetes
+CHART_VERSION_1 ?= local
```

Dont need the "_1" theres only one chart
```suggestion
CHART_VERSION ?= local
```